prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = 'Jennifer Noh';
+    console.log(fullName);
+
+    var numberOfPets = 0;
+    console.log(numberOfPets);
+
+    var likesVegetables = true;
+    console.log(likesVegetables);
+
+    var nothing = '';
```

Comment:
Hi Jennifer, for this variable we want to assign it the value `null` instead of an empty string.